scRNA-seq-advanced/setup/rms/config.yaml

```diff
@@ -0,0 +1,8 @@
+# The main directory where processed files will be stored
+base_dir: /shared/data/training-modules/scRNA-seq-advanced/data/rms
+
+# Libraries to process for the both the integration and differential gene expression modules
+libraries: ["SCPCL000478", "SCPCL000479", "SCPCL000480", "SCPCL000481", "SCPCL000483", "SCPCL000484", "SCPCL000488", "SCPCL000491", "SCPCL000495", "SCPCL000498"]
```

Comment:
YAMLify... pretty sure the quotes aren't needed, but you might check that
```suggestion
libraries: 
  - SCPCL000478
  - SCPCL000479
  - SCPCL000480
  - SCPCL000481
  - SCPCL000483
  - SCPCL000484
  - SCPCL000488
  - SCPCL000491
  - SCPCL000495
  - SCPCL000498
```

Reply:
confirmed syntax is ok